src/m365/planner/commands/plan/plan-details-get.ts

```diff
@@ -88,30 +88,19 @@ class PlannerPlanDetailsGetCommand extends GraphCommand {
       });
   }
 
-  private getPlanId(args: CommandArgs): Promise<string> {
+  private getPlanId(args: CommandArgs,logger: Logger): Promise<string> {
     if (args.options.planId) {
       return Promise.resolve(args.options.planId);
     }
 
-    const requestOptions: any = {
-      url: `${this.resource}/v1.0/groups/${this.groupId}/planner/plans`,
-      headers: {
-        accept: 'application/json;odata.metadata=none'
-      },
-      responseType: 'json'
-    };
-    
-    return request
-      .get<{ value: PlannerPlan [] }>(requestOptions)
-      .then((response: { value: PlannerPlan[] }): Promise<string> => {
-        const filteredPlan = response.value.filter((plan: PlannerPlan) => plan.title === args.options.planTitle);
+    return this.getAllItems(`${this.resource}/v1.0/groups/${this.groupId}/planner/plans`, logger, true, 'minimal')
```

Comment:
I don't think we need to specify the last argument as `minimal`. The default value of `none` is good, because we don't need any metadata in our code.